explorerlib/configBuilder.go

```diff
@@ -26,12 +26,6 @@ func (builder *botConfigurationBuilder) WithLanguages(languages []string) *botCo
 	return builder
 }
 
-func (builder *botConfigurationBuilder) AllAnonymous() *botConfigurationBuilder {
```

Comment:
You could keep the function even if you do not use it anymore?

Reply:
No, it wouldn't build.

Reply:
What was the error?
The existence of a function shouldn't prevent you from building :)